types/mapbox-gl/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Mapbox GL JS v0.54.0
+// Type definitions for Mapbox GL JS v1.3.0
```

Comment:
The PR title says it's for 1.4.0... Confusing.

Reply:
Yes, this is a bit confusing. There were no API changes in [v1.4.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.4.0) so these typings will work for v1.3.0 or v1.4.0. I can change the PR description if you think it makes more sense.

Reply:
Yes, plz do